scripts/mkcloud

```diff
@@ -376,11 +376,6 @@ function onhost_add_etchosts_entries
     grep -q crowbar /etc/hosts || echo "$adminip crowbar.$cloudfqdn crowbar" >> /etc/hosts
 }
 
-function onhost_enable_ksm
-{
-    ${mkclouddriver}_onhost_enable_ksm
```

Comment:
actually there is a `libvirt_enable_ksm` so when you drop the `onhost_` it could work

Reply:
@bmwiedemann the code is executed in the ```libvirt_do_prepare``` step. I added that to the commit msg. Is that enough?